src/cards/blood_curse.ts

```diff
@@ -82,6 +70,13 @@ const spell: Spell = {
     remove,
   },
   events: {
+    onTooltip: (unit: Unit.IUnit, underworld: Underworld) => {
+      const modifier = unit.modifiers[id];
+      if (modifier) {
+        // Set tooltip:
+        modifier.tooltip = `Blood Curse`;
+      }
+    },
```

Comment:
onTooltip is meant to run every gameLoop.  If the modifier is just the same as the id then it doesn't need to be set here because the id will be used instead.

Reply:
I think this was done a long time ago to prevent it from saying "Blood Curse 1"
It can be removed now?

Reply:
Oh let's leave it then